PSReadLine/PSReadLine.csproj

```diff
@@ -8,12 +8,12 @@
     <FileVersion>2.1.0</FileVersion>
     <InformationalVersion>2.1.0-beta2</InformationalVersion>
     <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
-    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
We could compile R2R if 3.1.

Reply:
The target 3.1 is for development only. For release, we only do 461, so it works across .NET and .NET Core